forms-flow-api/migrations/versions/e2b6b0b35d87_added_annonymous_field_fix_issues_fwf_.py

```diff
@@ -0,0 +1,30 @@
+"""added annonymous field & fix issues(FWF-1023) in Application & FormProcessMapper Table
+
+Revision ID: e2b6b0b35d87
+Revises: 166054bd81b5
+Create Date: 2022-01-18 04:32:02.043887
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "e2b6b0b35d87"
+down_revision = "166054bd81b5"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    ### commands auto generated by Alembic - please adjust! ###
+    op.add_column(
+        "form_process_mapper", sa.Column("is_anonymous", sa.Boolean(), nullable=True)
+    )
+    ### end Alembic commands ###
+
+
+def downgrade():
+    ### commands auto generated by Alembic - please adjust! ###
+    op.drop_column("form_process_mapper", "is_anonymous")
+    # ### end Alembic commands ###
```

Comment:
The original db migration file was edited to only include `is_anonymous` field. We need to have other field added to the next migration file in case late decided to fully migrate to later versions of forms-flow-ai